src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/IntermediateNodeWriter.cs

```diff
@@ -145,7 +145,7 @@ public override void VisitComponent(ComponentIntermediateNode node)
 
         public override void VisitComponentAttribute(ComponentAttributeIntermediateNode node)
         {
-            WriteContentNode(node, node.AttributeName, string.Format("AttributeStructure.{0}", node.AttributeStructure));
+            WriteContentNode(node, node.AttributeName, node.PropertyName, string.Format("AttributeStructure.{0}", node.AttributeStructure));
```

Comment:
Without this change it is hard to see in the IR baselines which parameter the current attribute belongs to. This has caused the bulk of changes in  `ir.txt` documents.